python/BUSINESS_SUPPORT_OR_ABOVE_ENABLED/parameters.json

```diff
@@ -0,0 +1,12 @@
+{
+  "Version": "1.0", 
+  "Parameters": {
+    "CodeKey": "business-support-enabled.zip", 
```

Comment:
Change to `BUSINESS_SUPPORT_OR_ABOVE_ENABLED`

Reply:
changed